runtime/reporter.rs

```diff
@@ -0,0 +1,84 @@
+use std::cell::RefCell;
+use std::fmt::Display;
+
+use serde_repr::Deserialize_repr;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Deserialize_repr)]
+#[repr(u8)]
+pub enum LogLevel {
+    Debug,
+    Info,
+    Warn,
+    Error,
+}
```

Comment:
`Deserialize_repr` allows us to parse numeric levels provided by Deno Console implementation and treat them as an enum in the rest of our codebase.